src/v/kafka/server/tests/fetch_session_test.cc

```diff
@@ -29,8 +30,7 @@ struct fixture {
     static kafka::fetch_session_partition make_fetch_partition(
       model::topic topic, model::partition_id p_id, model::offset offset) {
         return kafka::fetch_session_partition{
-          .topic = std::move(topic),
-          .partition = p_id,
+          .topic_partition = {topic, p_id},
```

Comment:
The changes to this test are just to accomodate the API (ntp -> ktp) changes elsewhere.